### ISSUE TITLE
Changing location of Taskbar does not work

### ISSUE THREAD

**Issue by fracz** [mwrock/boxstarter#88]
I have the following script:

```
Set-TaskbarOptions -Size Large -Lock -Dock Right
```

Icon size changes indeed (after logout/login). But the taskbar is still on bottom.

**Comment by gep13** [comment]
This is going to be a long shot.... But I have seen issues with this, when I first created this additional functionality in Boxstarter.  Can you try rebooting the machine after you have ran the script.  Sometimes the changes don't seem to "take" even though the Explorer instance gets rebooted.  I was never able to figure out why :-(

**Comment by fracz** [comment]
Neither icon size nor location changes immediately. Icon size is changed after relogin or reboot. I have to change location manually.

This is not a big deal. But I was excited it is possible to configure even such trivialities with Boxstarter :-)

**Comment by gep13** [comment]
Hmmm, that is annoying.  This certainly "used" to work.  If I get a chance, I will do some digging to see that I can find out.  Are the machines that you are trying this on connected to a domain, or anything like that?

**Comment by gep13** [comment]
So I have just tried this again on my test VM, and it also isn't working for me :-(

The original version of this was based on this article:

http://www.sevenforums.com/tutorials/1066-taskbar-move-location-desktop-screen.html

I have just tried the original registry keys, and they also aren't working, so I can only assume that something in Windows (via an update) has changed to make this no longer work :-(

This is really quite annoying, and not something that I have time to look into just now, but if I find some time, I will certainly take another stab at it.

@mwrock any thoughts on what might have changed in Windows to prevent this from working now?

**Comment by mwrock** [comment]
No, but I'll definitely be taking a look at this and report back.

**Comment by gep13** [comment]
:+1:

**Comment by dhilgarth** [comment]
Changing the size still works in Windows 10. Changing the dock didn't work correctly

**Comment by gep13** [comment]
@dhilgarth yeah, this is really unfortunate.  Looks like there is no reliable way to change the dock position, which is really annoying.  The original method definitely used to work :cry:
